packages/dds/tree/src/shared-tree/sharedTree.ts

```diff
@@ -110,6 +111,24 @@ export interface ISharedTree extends ISharedObject, ITree {
 	): FlexTreeView<TRoot> | undefined;
 }
 
+interface TopLevelCodecVersions {
```

Comment:
Looks like only `editManager` is being leveraged right now. Is that just because this is WIP?

Reply:
Yep.